tests/terraform/checks/resource/aws/test_CloudwatchLogGroupRetention.py

```diff
@@ -0,0 +1,27 @@
+import unittest
+
+from checkov.terraform.checks.resource.aws.cloudwatchLogGroupRetention import check
+from checkov.common.models.enums import CheckResult
+
+
+class TestCloudwatchLogGroupRetention(unittest.TestCase):
+
+    def test_failure(self):
+        resource_conf = {
```

Comment:
@jroden To make tests which represent real life, try following [this](https://github.com/bridgecrewio/checkov/blob/e5560e734f5328c2b9f8abd70e531dade21f20a0/tests/terraform/checks/resource/aws/test_SecurityGroupUnrestrictedIngress22.py#L12) pattern.